examples/pytorch/model_zoo/classification.py

```diff
@@ -0,0 +1,87 @@
+from dgl.data import Tox21
+from dgl.data.utils import split_dataset
+from dgl import model_zoo
+import torch
+from torch.nn import BCEWithLogitsLoss
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from utils import Meter, EarlyStopping, collate_molgraphs
+
+def main():
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    batch_size = 128
+    learning_rate = 0.001
+    num_epochs = 100
```

Comment:
Unless you've re-tuned the model to match the performance and updated the pretrained model, I'd suggest changing this back to my old configurations to ensure reproducible performance.

Reply:
This is an use case demonstration, user should not use it with arguments. 
Reproducing doesn't work with gpu, I don't want to introduce the complexity here.